src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.DesignerSupport.targets

```diff
@@ -139,6 +139,13 @@ Copyright (c) .NET Foundation. All rights reserved.
         Include="@(_DesignerShadowCopy->'%(FullPath)')"
         TargetPath="%(_DesignerShadowCopy.DestinationSubDirectory)%(_DesignerShadowCopy.Filename)%(_DesignerShadowCopy.Extension)"
         />
+
+      <!-- Include ijwhost.dll for projects that use it -->
+      <DesignerRuntimeImplementationProjectOutputGroupOutput
```

Comment:
It seems like this target would need to depend on `ResolveFrameworkReferences` in order to correctly have the `IjwHostSourcePath` set.

Reply:
From locally building, it does have that via a dependency chain of `DesignerRuntimeImplementationProjectOutputGroup` > `AssignTargetPaths` > `_ComputeNETCoreBuildOutputFiles` > `_GetIjwHostPaths` > `ResolveFrameworkReferences`